snippets/htmx.md

```diff
@@ -6,22 +6,33 @@ Using Hono with [htmx](https://htmx.org/).
 
 ### typed-htmx
 
-With using [typed-htmx](https://github.com/Desdaemon/typed-htmx), you can write the JSX with the type definitions of htmx.
+By using [typed-htmx](https://github.com/Desdaemon/typed-htmx), you can write JSX with TypeScript definitions for htmx attributes.
+We can follow the same pattern found on the [typed-htmx Example Project](https://github.com/Desdaemon/typed-htmx/blob/main/example/src/types.d.ts) to use it with `hono/jsx`.
 
 Install:
 
 ```sh
 npm i -D typed-htmx
 ```
 
-`tsconfig.json`:
+`app/global.d.ts`:
```

Comment:
Some users will use `src` and have to make `src/global.d.ts`. So you can add a note about it.

Reply:
Is the `app` folder part of the honox template exclusively?
If so, maybe something like the following?

```md
`src/global.d.ts` (or `app/global.d.ts`, if you're using HonoX):
```

Reply:
Yeah. Exactly!

Reply:
Done!